src/apps/statements/repo/modelsRepo/getStatements/mongo/index.ts

```diff
@@ -32,7 +32,7 @@ const filterModels = (opts: Opts): Object => {
 const sortModels = (ascending: boolean) => {
   return {
     stored: ascending ? 1 : -1,
-    _id: -1,
+    _id: ascending ? 1 : -1,
```

Comment:
@ryansmith94 , ![](https://media1.tenor.com/images/65866ba370141523c412fb6c0dd2de6d/tenor.gif?itemid=4937950)

How about so? 😃

Reply:
Yeah I think that will solve that first problem that is reproducible via API requests. The second problem will involve changing the construction and use of the cursor to also include the stored value of the last item retrieved. I'll try to come up with a way to reproduce it.